.github/workflows/wheels-linux.yml

```diff
@@ -0,0 +1,55 @@
+# Build, test and publish wheels to PyPI
+# Build and test run on every push
+# Publish runs only on a tag push
+# Note: This does not publish sdist
+name: Wheels (Linux)
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build wheels on Linux
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - uses: actions/setup-python@v2
+      name: Install Python
+      with:
+        python-version: 3.8
```

Comment:
Hi, the current Travis CI builds on 3.5, 3.6, 3.7, 3.8 and 3.9. Can Github Actions do the same? Thanks!

Reply:
This is just the base image's python version. Actual wheels are built for 3.5, 3.6, 3.7, 3.8 transparently by cibuildwheel

Refer: https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip